doc/site/content/docs/guides/rendering/making-shaders.md

```diff
@@ -0,0 +1,366 @@
++++
+title = "Making your own shaders"
+author = "Slashscreen"
++++
+
+Recoil comes with some premade shaders that cover the majority of use cases, but there are some occasions in which you need custom shaders. Luckily, Recoil lets you to just about whatever you want. For the uninitiated, though, it can be a little confusing; that's where this guide comes in. 
+
+## Step 1: The Shader
+
+First thing's first, let's make a basic shader.
+
+**Vertex Shader (`shader.vert.glsl`):**
+```glsl
+#version 460
```

Comment:
You might want to leave the note it's a GL4 class shader. The version setting will rule out older GPUs / drivers